main/solution/backend/config/infra/cloudformation.yml

```diff
@@ -770,6 +770,7 @@ Resources:
               - 'arn:aws:iam::*:role/*-cross-account-role' # This is required since when users onboard the main account as a member account, the onboard CFN stack would also reside there
               - 'arn:aws:iam::*:role/*-xacc-env-mgmt'
               - !Sub 'arn:aws:iam::${AWS::AccountId}:role/${self:custom.settings.namespace}-prep-raas-master-MasterRole-*'
+              - 'arn:aws:iam::*:role/${self:custom.settings.globalNamespace}-DevopsRole'
```

Comment:
Can we add this resource value only if `enableAmiSharing` is set to `true`? [Example](https://github.com/awslabs/service-workbench-on-aws/blob/c6dd022200fde668476f5aee247f2383d8d54bf2/main/solution/backend/config/infra/cloudformation.yml#L3)